app/Models/Signup.php

```diff
@@ -11,7 +11,7 @@ class Signup extends Model
      *
      * @var array
      */
-    protected $fillable = ['id', 'event_id', 'northstar_id', 'campaign_id', 'campaign_run_id', 'quantity', 'quantity_pending', 'why_participated', 'source', 'created_at', 'updated_at'];
```

Comment:
sorry if I missed this! Why are we getting rid of the `event_id` in this table?

Reply:
ah! it's not actually still in the table so I just removed it since I noticed.

Reply:
ooo i think `event_id` was left over here from a past iteration of the DB schema when we had everything tied to an event

Reply:
ah right right - thank you both!